src/DeploymentManager/DeploymentManager/Commands/RemoveService.cs

```diff
@@ -0,0 +1,187 @@
+        [ResourceGroupCompleter]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true, 
+            ParameterSetName = DeploymentManagerBaseCmdlet.InteractiveParamSetName,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The name of the service topology.")]
+        [ValidateNotNullOrEmpty]
+        public string ServiceTopologyName { get; set; }
+
+        [Parameter(
+            Position = 2,
+            Mandatory = true, 
+            ParameterSetName = DeploymentManagerBaseCmdlet.InteractiveParamSetName,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The name of the service.")]
+        [Parameter(
+            Position = 1,
+            Mandatory = true, 
+            ParameterSetName = RemoveService.ByServiceTopologyObjectParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The name of the service.")]
+        [Parameter(
+            Position = 1,
+            Mandatory = true, 
+            ParameterSetName = RemoveService.ByServiceTopologyResourceIdParamSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The name of the service.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true, 
+            ParameterSetName = DeploymentManagerBaseCmdlet.ResourceIdParamSetName,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource identifier.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true, 
+            ParameterSetName = DeploymentManagerBaseCmdlet.InputObjectParamSetName,
+            ValueFromPipeline = true,
+            HelpMessage = "Service object.")]
+        [ValidateNotNullOrEmpty]
+        public PSServiceResource Service { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true, 
+            ParameterSetName = RemoveService.ByServiceTopologyObjectParameterSet,
+            HelpMessage = "The service topology object in which the service should be created.")]
+        [ValidateNotNullOrEmpty]
+        public PSServiceTopologyResource ServiceTopology { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true, 
+            ParameterSetName = RemoveService.ByServiceTopologyResourceIdParamSet,
+            HelpMessage = "The service topology resource identifier in which the service should be created.")]
+        [ValidateNotNullOrEmpty]
+        public string ServiceTopologyResourceId { get; set; }
+
+        [Parameter(
+            Mandatory = false, 
+            HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
```

Comment:
@deveshguha same comment about removing this parameter if it's not necessary

Reply:
Was made to add these in the earlier version as these are destructive operations.

Reply:
@deveshguha same comment as above

Reply:
Removed